src/components/structures/MatrixChat.tsx

```diff
@@ -393,6 +394,8 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         PosthogAnalytics.instance.updatePlatformSuperProperties();
 
         CountlyAnalytics.instance.enable(/* anonymous = */ true);
+
+        initSentry(SdkConfig.get()["sentry"]);
```

Comment:
Needs documenting in https://github.com/vector-im/element-web/blob/develop/docs/config.md

Reply:
https://github.com/vector-im/element-web/pull/18608